app/controllers/browse_controller.rb

```diff
@@ -15,7 +15,29 @@ def show
 
     respond_to do |f|
       f.html do
-        render :show, locals: { page: page }
+        is_page_under_ab_test = false
+
+        dimension = Rails.application.config.navigation_ab_test_dimension
+        ab_test = GovukAbTesting::AbTest.new("EducationNavigation", dimension: dimension)
+        ab_test_variant = ab_test.requested_variant(request)
+
+        if new_navigation_enabled? && params[:top_level_slug] == "education"
```

Comment:
Will this direct all child browse pages of education to the root of the new taxonomy?